common/src/commonMain/kotlin/Platform.kt

```diff
@@ -0,0 +1,10 @@
+package dev.kord.common
+
+import dev.kord.common.annotation.KordInternal
+
+@KordInternal
+public expect object Platform {
```

Comment:
should we move this to `test-kit` while we don't need it in actual kord code?

this should eliminate these warnings (they seem to come up because of the dependency on `projects.common` in `test-kit` for access to this object):
```
> Task :common:compileTestKotlinJs                                                                                                                                                                                                                                                                                  
w: duplicate library name: kord:common 

> Task :common:compileTestDevelopmentExecutableKotlinJs                                                                                                                                                                                                                                                             
w: duplicate library name: kord:common
```